### ISSUE TITLE
Download all playlists from a channel and sort/filter them whether they are music or video.

### ISSUE THREAD

**Issue by falzard** [rg3/youtube-dl#11668]
### What is the purpose of your *issue*?
- [ ] Bug report (encountered problems with youtube-dl)
- [ ] Site support request (request for adding support for a new site)
- [ ] Feature request (request for a new functionality)
- [X] Question
- [ ] Other
---

### Description of your *issue*, suggested solution and other information

I found a command to download an entire channel and sort the channel by playlists into directories: %(uploader)s/%(playlist)s/%(playlist_index)s - %(title)s.%(ext)s  (part of the command)

I'd like to download my own playlists which are either audio or video playlists, and **I'd like to automatically convert to audio the audio playlists but still download the video playlists as videos in one command**.
So basically I'd simply run my command and download the whole channel each time I add new videos to my playlists.

Someone said to try with --match-filter but I have no idea how. There must be a trick like appending to each audio playlists "[audio]" and then filter them but I'm not experienced in bash and in youtube-dl.

How to do so?

**Comment by dstftw** [comment]
Not possible.
If it's your playlists just add `[audio]` or `[video]` to corresponding playlists' names and they will be extracted with `%(playlist)s`.

**Comment by falzard** [comment]
If I add [audio] and [video], what will it do? i don't get it
It won't convert to audio the audio playlists and keep the video playlist as video, right?
